lib/api/apiUtils/object/versioning.js

```diff
@@ -98,6 +98,7 @@ function _storeNullVersionMD(bucketName, objKey, nullVersionId, objMD, log, cb)
             isNull2: true,
         });
     }
+    nullVersionMD.originOp = 's3:ObjectNullVersionStoredAsNewVersion';
```

Comment:
maybe something shorter: `StoreNullVersion` ?
(minimal optimization, but anything we add ends up pilling in metadata.... and with billions of objects, it quickly adds up)